modules/components/org.wso2.carbon.appfactory.core/src/main/java/org/wso2/carbon/appfactory/core/sql/SQLConstants.java

```diff
@@ -90,6 +102,23 @@
     public static final String GET_APPLICATION_CREATION_STATUS_SQL = "SELECT STATUS FROM " +
             "AF_APPLICATION WHERE APPLICATION_KEY=? AND TENANT_ID=?";
 
+    public static final String GET_DEPLOYEMENET = "SELECT DEPLOYMENT_ID, DEPLOYMENT_NAME, REPLICAS FROM AF_DEPLOYMENT "
+            + "WHERE DEPLOYMENT_NAME = ?";
+
+    public static final String GET_CONTAINER = "SELECT CONTAINER_ID, BASEIMAGE_NAME, BASEIMAGE_VERSION FROM AF_CONTAINER"
+            + " WHERE DEPLOYMENT_ID = ?";
+
+    public static final String GET_SERVICE_PROXY = "SELECT SERVICE_NAME, SERVICE_PROTOCOL, SERVICE_PORT, "
+            + "SERVICE_BACKEND_PORT FROM AF_SERVICE_PROXY WHERE CONTAINER_ID = ?";
+
+    public static final String GET_DEPLOYEMENT_CONFIG = "SELECT deployment.DEPLOYMENT_NAME, deployment.REPLICAS,"
+            + "container.BASEIMAGE_NAME, container.BASEIMAGE_VERSION,"
+            + "service.SERVICE_NAME, service.SERVICE_PROTOCOL, service.SERVICE_PORT, service.SERVICE_BACKEND_PORT"
+            + "FROM AF_DEPLOYMENT deployment"
+            + "INNER JOIN AF_CONTAINER container ON deployment.DEPLOYMENT_ID = container.DEPLOYMENT_ID"
```

Comment:
This is fixed in following commit.
https://github.com/wso2/product-af/pull/800/files#diff-15378ae9b07f2a36a5805111f6307d9bR114